src/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/MemberServiceTests.cs

```diff
@@ -34,7 +34,7 @@ public class MemberServiceTests : UmbracoIntegrationTest
         public void SetupTest()
         {
             // TODO: remove this once IPublishedSnapShotService has been implemented with nucache.
-            Core.Services.Implement.MemberTypeService.ClearScopeEvents();
+            global::Umbraco.Core.Services.Implement.MemberTypeService.ClearScopeEvents();
```

Comment:
This ⬇️ would work if you do `private MemberTypeService MemberTypeService => (MemberTypeService) GetRequiredService<IMemberTypeService>();` at the beginning of the file

```suggestion
            MemberTypeService.ClearScopeEvents();
```

Reply:
See above.